site/content/_index.md

```diff
@@ -15,3 +35,17 @@ graph TD;
 ## Resources
 * [Glossary](glossary)
 * [References and Related Work](refs-related)
+
+## Why code reviews?
+
+Code review is standard practice in the software industry, as well as
+open source software communities. It is, however, very rare in
+academia.  Very generally, we believe that developers of research
```

Comment:
Can we say something like "currently it is not commonly practiced in academia" -- a little less aggressive